docs/docusaurus/docs/conceptual_guides/gx_overview.md

```diff
@@ -0,0 +1,148 @@
+
+## Great Expectations (GX)
+
+GX is a Python library that provides a framework for describing the acceptable state of data and then validating that the data meets those criteria. 
+
+GX enables you to build trust in your data's quality by testing how it performs against your expectations.  Reusable components and ongoing testing provide visibility into the health of your data pipeline over time. The human-readable reports generated by GX allow you to share the results of that analysis with others and better collaborate on improvements to both your data and your understanding of your data.
+
+With GX providing insight into your data and helping assure its quality, you can ensure that your data is fully suitable to be used as intended in your processes.
+
+## GX core components
+
+When working with GX from end to end - that is, from installation of GX to the validation of your data - there are four core components that you will use.  These are "top level" components that interface, contain, or manage underlying objects and processes to make your work easier.  This guide will go over each of these four components and what they govern in more depth after this section. However, in brief, the four components are:
+
+- **Data Context:** Manages the settings and metadata for a GX project, as well as providing an entry point to the GX Python API.
+- **Datasources:** tell GX how to connect to your source data, and how to organize the retrieved data for future use.
+- **Expectations:** tell GX the standards to which your data should conform.
+- **Checkpoints:** validate a set of Expectations against a specific set of data.
+
+
+## Data Context
+
+![A Data Context](./images_overview/data_context_flowchart.png)
+
+A Data Context defines a GX project.  In Python, the Data Context object serves as your entry point for the GX API and manages various classes to limit the objects you need to directly manage yourself.  A Data Context contains all of the metadata used by GX, the configurations for GX objects, and the output from validating data.
+
+The following are the available Data Context types:
+- **Ephemeral Data Context:** Exists in memory, and does not persist beyond the current Python session.
+- **File Data Context:** Exists as a folder and configuration files. Its contents persist between Python sessions.
+- **Cloud Data Context:** Supports persistence between Python sessions, but additionally serves as the entry point for Great Expectations Cloud.
+
+- [Data Context tutorials](/docs/guides/setup/configure_data_contexts_lp)
```

Comment:
The linked topic isn't a tutorial.

I would remove this as bullet point from the list. Instead, introduce the information. For example:

To instantiate and convert a Data Context, see [Configure Data Contexts](/docs/guides/setup/configure_data_contexts_lp).